plugin/src/main/scala/com/typesafe/genjavadoc/AST.scala

```diff
@@ -97,14 +97,16 @@ trait AST { this: TransformCake ⇒
     if (m.isPublic) "public"
     else if (m.isProtected && !topLevel) "protected"
     else if (m.isPrivate && !topLevel) "private"
-    else "public" // this is the case for “private[xy]” and top level classes
+    else if (m.privateWithin != tpnme.EMPTY) ""
```

Comment:
`private[org] val u` under `org.example` has to be public in Java. Otherwise, if there exists references to `u` under `org`, the Java code won't compile. Is it correct?